docs/semgrep-appsec-platform/webhooks.md

```diff
@@ -14,6 +14,10 @@ import Notifications from "/src/components/concept/_notification-deduplication.m
 
 Webhooks are a generic method for Semgrep AppSec Platform to post JSON-formatted findings after each scan to your URL endpoint.
 
+:::tip For Slack integrations
+- To integrate with Slack, use the [Semgrep Slack integration](/semgrep-appsec-platform/slack-notifications). The webhook set up described in this guide does not apply to Slack integrations.
```

Comment:
I thought of writing "does not work for Slack integrations" but was worried it was too strong a phrase.

Reply:
It's accurate, though! It literally does not work when you try to set it up. I leave this to your discretion.